src/crypto/variant4_random_math.h

```diff
@@ -0,0 +1,418 @@
+#ifndef VARIANT4_RANDOM_MATH_H
+#define VARIANT4_RANDOM_MATH_H
+
+// Register size can be configured to either 32 bit (uint32_t) or 64 bit (uint64_t)
+typedef uint32_t v4_reg;
+
+enum V4_Settings
+{
+	// Generate code with minimal theoretical latency = 45 cycles, which is equivalent to 15 multiplications
+	TOTAL_LATENCY = 15 * 3,
+	
+	// Always generate at least 60 instructions
+	NUM_INSTRUCTIONS = 60,
+	
+	// Available ALUs for MUL
+	// Modern CPUs typically have only 1 ALU which can do multiplications
+	ALU_COUNT_MUL = 1,
+
+	// Total available ALUs
+	// Modern CPUs have 4 ALUs, but we use only 3 because random math executes together with other main loop code
+	ALU_COUNT = 3,
+};
+
+enum V4_InstructionList
+{
+	MUL,	// a*b
+	ADD,	// a+b + C, -128 <= C <= 127
```

Comment:
`-128 <= C <= 127` means C is an 8-bit constant. Though it is declared and used as a 32-bit unsigned.
https://github.com/monero-project/monero/pull/5126/files#diff-841dcad45a1a1f90a44e2f76be84872fR56
https://github.com/monero-project/monero/pull/5126/files#diff-841dcad45a1a1f90a44e2f76be84872fR364

Reply:
Yes, this is an old comment. It's actually a 32-bit constant.